tbx/core/blocks.py

```diff
@@ -100,6 +102,28 @@ class Meta:
 class PullQuoteBlock(StructBlock):
     quote = CharBlock(form_classname="quote title")
     attribution = CharBlock()
+    company_logo = ImageChooserBlock(
+        required=False, help_text="Add either a company logo or author image"
+    )
+    author_image = ImageChooserBlock(
+        required=False, help_text="Add either a company logo or author image"
```

Comment:
**Question**: This seems to suggest that the content editor *should* add either the logo or author image. However, not adding either is also an option. Is there a way the wording can be adjusted to reflect this?